src/runtime/uv/event_loop.cpp

```diff
@@ -98,13 +98,15 @@ extern "C" LEAN_EXPORT lean_obj_res lean_uv_event_loop_configure(b_obj_arg optio
 
     event_loop_lock(&global_ev);
 
-    if (accum && uv_loop_configure(global_ev.loop, UV_METRICS_IDLE_TIME) != 0) {
+    int result = uv_loop_configure(global_ev.loop, UV_METRICS_IDLE_TIME);
+    if (accum && result != 0) {
         return io_result_mk_error("failed to configure global_ev.loop with UV_METRICS_IDLE_TIME");
```

Comment:
Is there are a reason why you're not calling `lean_decode_uv_error` here?

Reply:
No, I just forgot. The behavior of the function changed too and I fixed it.